_includes/head.html

```diff
@@ -1,13 +1,50 @@
+{% if page.title %}
+  {% assign title = page.title %}
```

Comment:
This changes the titles of pages to remove the `| Student Robotics` from pages. Is that intentional?

Reply:
Originally removed it to make the OpenGraph title the same as the page title but on further thought there is no reason these need to match so have added back 223b255